components/compositing/compositor.rs

```diff
@@ -1483,7 +1483,7 @@ impl<Window: WindowMethods> IOCompositor<Window> {
             _ => initialize_png(&*self.gl, width, height)
         };
 
-        profile(ProfilerCategory::Compositing, None, self.time_profiler_chan.clone(), || {
+        profile(ProfilerCategory::Compositing, None, &self.time_profiler_chan.clone(), || {
```

Comment:
Ah, I see. In that case, let's revert the change to profile - I suspect many people will make the same assumption that I did (that & followed by clone() is unnecessary) and run into this same error.